docs/cloud-native-security/cspm-get-started-azure.asciidoc

```diff
@@ -83,39 +90,80 @@ After assigning the role:
+
+. On the **Add Cloud Security Posture Management (CSPM) integration** page, under **Setup access**, select **Manual**. 
+. Under **Preferred manual method**, select **Service principal with client certificate**.
+. Go to the **Registered apps** section of https://ms.portal.azure.com/#view/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/~/RegisteredApps[Microsoft Entra ID].
+. Click on **New Registration**, name your app and click **Register**.
+. Copy your new app's `Directory (tenant) ID` and `Application (client) ID`. Paste them into the corresponding fields in {kib}.
+. Return to Azure. Go to your Azure subscription list and select the subscription or management group you want to monitor with CSPM.
+. Go to **Access control (IAM)** and select **Add Role Assignment**. 
+. Select the `Reader` function role, assign access to **User, group, or service principal**, and select your new app.
+
+Next, create a certificate. If you intend to use a password-protected certificate, you must use a pkcs12 certificate. Otherwise, you must use a pem certificate.
+
+Create a pkcs12 certificate, for example:
+```shell
+# Create PEM file
+openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes
+
+# Create pkcs12 bundle using legacy flag (CLI will ask for export password)
+openssl pkcs12 -legacy -export -out bundle.p12 -inkey key.pem -in cert.pem
 ```
-AZURE_TENANT_ID=<Directory (tenant) ID>
-AZURE_CLIENT_ID=<Application (client) ID>
-AZURE_CLIENT_SECRET=<Secret Value>
+
+Create a PEM certificate, for example:
+```shell
+# Generate certificate signing request (csr) and key
+openssl req -new -newkey rsa:4096 -nodes -keyout cert.key -out cert.csr
+
+# Generate PEM and self-sign with key
+openssl x509 -req -sha256 -days 365 -in cert.csr -signkey cert.key -out signed.pem
+
+# Create bundle
+cat cert.key > bundle.pem
+cat signed.pem >> bundle.pem
 ```
 
-After creating the file:
+After creating your certificate:
 
+. Return to Azure.
+. Navigate to the **Certificates & secrets** menu. Select the **Certificates** tab.
+. Click **Upload certificate**. 
+.. If you're using a PEM certiciate created using the example commands above, upload `signed.pem`.
+.. If you're using a pkcs12 certificate created using the example commands above, upload `cert.pem`.
+. Upload the certificate bundle to the VM where you will deploy {agent}.
+.. If you're using a PEM certificate created using the example commands above, upload `bundle.pem`.
+.. If you're using a pkcs12 certificate created using the example commands above, upload `bundle.p12`.
 . Return to the **Add CSPM** page in {kib}. 
-. Under **Configure integration**, select **Azure**. Under **Setup access**, select **Manual**.
+. For **Client Certificate Path**, enter the fullpath to the certificate you uploaded to the host where you will install {agent}. 
```

Comment:
I'm not sure if fullpath should be one word, so leaving an edit here in case it shouldn't. Also suggesting an optional "that" to break up the sentence a bit.

```suggestion
. For **Client Certificate Path**, enter the full path to the certificate that you uploaded to the host where you will install {agent}. 
```